indicators/NO_GLAC_001/R/NO_GLAC_001.qmd

```diff
@@ -0,0 +1,1731 @@
+distributions, which can be sampled.
+
+```{r tbl-normalise}
+#| tbl-cap: 'Table of ecosystem assets, with variable values, X100 values (glacier area 1947-1985), indicator values and quartiles.'
+#| code-fold: true
+#| code-summary: "Normalise variable"
+
+
+# create a temporary dataframe
+EAtemp <- EA |>
+  as_tibble() |>
+  # unlist and create new rows for each random pair of current and old glacier area estimate
+  unnest_longer(c(
+    glacierAreaCurrent_dist,
+    glacierAreaOld_dist), 
+    keep_empty = T) |>
+  rename(variable = glacierAreaCurrent_dist, 
+         x100 = glacierAreaOld_dist) |>
+  mutate(
+    # replace NA with zero if EA had glaciers, but lost them
+    variable = case_when(
+      is.na(variable) & x100 > 0 ~ 0,
+      .default = variable),
+    # define x60 value
+    x60 = x100*0.1,
+    # normalise using piecewise linear transfomation
+    indicator =  ifelse(
+      variable < x60, 
+        ((variable - 0)/(x60 - 0)) * 0.6, 
+        ((variable - x60)/(x100 - x60)) * (1 - 0.6) + 0.6),
+    # truncate
+    indicator = case_when(
+      indicator > 1 ~ 1,
+      indicator < 0 ~ 0,
+      .default = indicator
+    )) |>
+  # recast into list column
+  drop_na(indicator) |>
+  group_by(EA_ID) |>
+  summarise(indicator_dist = list(indicator), .groups = "drop")
+
+# we then paste these distributions back into the EA object
+
+EA <- EA |>
+  as_tibble() |>
+  # simplify datafra by removing some columns...
+  select(-glacierAreaCurrent_dist,
+         -glacierAreaOld_dist,
+         -GID_0,
+         -NAME_0) |>
+  # add the probability distribution of indicator values
+  left_join(
+    EAtemp,
+    by = join_by(EA_ID)
+  ) |>  
+  # summarise the distribution
+  rowwise() |>
+  mutate(
+    indicator = ifelse(
+      !is.null(indicator_dist),
+      median(unlist(indicator_dist)),
+      NA),
+  # add first and third quartile
+    indicator_Q1 = ifelse(
+      !is.null(indicator_dist),
+      quantile(unlist(indicator_dist), probs = 0.25),
+      NA), 
+    indicator_Q3 = ifelse(
+      !is.null(indicator_dist),
+      quantile(unlist(indicator_dist), probs = 0.75),
+      NA), 
+  ) |>
+  st_as_sf()
+
+DT::datatable(EA |>
+  as_tibble() |>
+  mutate(across(where(is.numeric), ~ round(.x, digits = 2))) |>
+  select(
+    EA_ID,
+    glacierAreaCurrent,
+    X100 = glacierAreaOld,
+    mountainArea,
+    indicator,
+    indicator_Q1,
+    indicator_Q3
+  ))
+```
+
+@fig-EAglacArea show the current (2018-2019) glacier extent per EA,
+stratified by bioclimatic sone and region. Clearly most glaciers are in section 2, 
+which is the _clearly oceanic section_. High percipitation is a prerequitese for glaciers. 
+Section 1 has even more precipitation, but is also warmer and has less total extent (@fig-sonerOgSeksjoner).
+
+Of the sones, its mainly in the alpine sones (sone 5) where we find glaciers. 
+@fig-gradientIndicator show how the warmer sones also have a little lower indicator values,
+as expected.
```

Comment:
Why do the glaciers in sone 3 not have any extent? Are they all lost? And why is their indicator value not much lower then or 0? Or am I missing something here?

Reply:
In @fig-EAglacArea we see there are no glaciers in zone 3, but that was just because the values were so small they didnt show. I turned to a log10 axis to avoid that kind of confusion. 

The indicator values for zone 3 are generally quite low, but they are not zero unless they are all lost (one case of that happening)